src/injected/styles/injected.scss

```diff
@@ -266,14 +267,14 @@
         text-align: center !important;
     }
 
-    .insights-highlight-box {
+    .insights-highlight-container .insights-highlight-box {
```

Comment:
Scoping on lines 270 and 277 weren't required but I added them to keep the styling consistent. I confirmed that this didn't create a regression of the styles that #4253 addressed.